src/views/AuthoritiesSearch/AuthoritiesSearch.js

```diff
@@ -192,9 +223,55 @@ const AuthoritiesSearch = ({
     ...options,
   ];
 
+  const getActionItem = ({ id, icon, messageId, onClickHandler, isDisabled = false }) => {
+    return (
+      <Button
+        buttonStyle="dropdownItem"
+        id={id}
+        disabled={isDisabled}
+        onClick={onClickHandler}
+      >
+        <Icon
+          icon={icon}
+          size="medium"
+          iconClassName={css.actionIcon}
+        />
+        <FormattedMessage id={messageId} />
+      </Button>
+    );
+  };
+
+  const triggerQuickExport = async (sendCallout) => {
```

Comment:
Let's create a hook out of this - we should split code as much as possible
Call it `useAuthorityExport`. Also, we need to make a request to back-end to start an export job.
In `InstancesList` there's an example:
```
await this.props.parentMutator.quickExport.POST({
        uuids: instanceIds,
        type: 'uuid',
        recordType: 'INSTANCE'
      });
 ```
 It uses mutators to make a request, and `ui-marc-authorities` uses `react-query` so we can't copy-paste it, but essentially what it does is sends a POST request to `data-export/quick-export` with body 
 ```
 {
        uuids: instanceIds,
        type: 'uuid',
        recordType: 'INSTANCE'
      }
 ```
 We need to change `INSTANCE` to `AUTHORITY`